src/svg.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const markup: string;
+  export default markup;
+}
```

Comment:
Actually, I'm wondering if we might want to do this now. I had seen this PR a few days ago—sorry for the delay. I paused because I wanted to make sure that what is going on here is clear to future devs. Sticking this in a `types` directory might be good signposting...what do you think?

Reply:
Sounds reasonable to me. I've pushed a commit to move this file into `src/types/`.